models/pancreas_ct_dints_segmentation/configs/metadata.json

```diff
@@ -0,0 +1,80 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.1.0",
+    "changelog": {
+        "0.1.0": "complete the model package",
+        "0.0.1": "initialize the model package structure"
+    },
+    "monai_version": "0.9.0",
+    "pytorch_version": "1.10.0",
+    "numpy_version": "1.21.2",
+    "optional_packages_version": {
+        "fire": "0.4.0",
+        "nibabel": "3.2.1",
+        "pytorch-ignite": "0.4.9"
+    },
+    "task": "Neural architecture search on pancreas CT segmentation",
+    "description": "Searched architectures for volumetric (3D) segmentation of the pancreas from CT image",
+    "authors": "MONAI team",
+    "copyright": "Copyright (c) MONAI Consortium",
+    "data_source": "Task07_Pancreas.tar from http://medicaldecathlon.com/",
+    "data_type": "nibabel",
+    "image_classes": "single channel data, intensity scaled to [0, 1]",
+    "label_classes": "single channel data, 1 is pancreas, 2 is pancreatic tumor, 0 is everything else",
+    "pred_classes": "3 channels OneHot data, channel 1 is pancreas, channel 2 is pancreatic tumor, channel 0 is background",
+    "eval_metrics": {
+        "mean_dice": 0.65
```

Comment:
Hi Dong, is the score here achieved from `evaluate.yaml`?
I checked with the weights you provided in `large_files.yml`, but got a different result:
```
2022-06-22 06:50:54,726 - ignite.engine.engine.SupervisedEvaluator - INFO - Got new best metric of val_mean_dice: 0.7196152806282043
2022-06-22 06:50:54,726 - ignite.engine.engine.SupervisedEvaluator - INFO - Epoch[1] Metrics -- val_accuracy: 0.9994 val_mean_dice: 0.7196
2022-06-22 06:50:54,727 - ignite.engine.engine.SupervisedEvaluator - INFO - Key metric: val_mean_dice best value: 0.7196152806282043 at epoch: 1
2022-06-22 06:50:54,735 - ignite.engine.engine.SupervisedEvaluator - INFO - Epoch[1] Complete. Time taken: 00:40:18
2022-06-22 06:50:54,735 - ignite.engine.engine.SupervisedEvaluator - INFO - Engine run complete. Time taken: 00:40:18
```